src/components/Send.jsx

```diff
@@ -286,7 +296,19 @@ export default function Send() {
               title="Send bitcoin"
               subtitle="Collaborative transactions increase the privacy of yourself and others."
             />
+
+            <rb.Fade in={!isCoinjoinOptionEnabled} mountOnEnter={true} unmountOnExit={true}>
+              <div className="mb-4 border border-1 p-3" style={{ borderRadius: '.25rem' }}>
+                🛈{` `}
```

Comment:
Same comments as in #82. 😉

Reply:
In this case `npm run format` will insert `{' '}`.